src/comment.rs

```diff
@@ -71,6 +71,16 @@ pub fn rewrite_comment(orig: &str,
     let indent_str = offset.to_string(config);
     let line_breaks = s.chars().filter(|&c| c == '\n').count();
 
+    let num_bare_lines = s.lines()
+        .enumerate()
+        .map(|(_, line)| line.trim())
+        .filter(|l| !(l.starts_with('*') || l.starts_with("//") || l.starts_with("/*")))
+        .count();
+
+    if num_bare_lines > 0 && !config.normalize_comments {
```

Comment:
Bailing early would mean that comments like found in `comment4.rs` where the comment starts at wrong indentation levels would not be fixed indentation wise.

I at least want comments to be aligned when they only have non bare lines, even though I don't want the other things that `normalize_comments` would normally do.

The `normalize_comments` setting imo is more about the internal layout of the comments, and not about alignment, as alignment only moves the comments around.

Should I add a new option? Maybe calling it `align_comments`? Or should I convert normalize_comments to an enum?